src/target/riscv/riscv-013.c

```diff
@@ -4459,8 +4424,10 @@ riscv_insn_t riscv013_read_debug_buffer(struct target *target, unsigned index)
 int riscv013_invalidate_cached_debug_buffer(struct target *target)
 {
 	dm013_info_t *dm = get_dm(target);
-	if (!dm)
+	if (!dm) {
+		LOG_ERROR("No DM is specifyed for the target");
```

Comment:
Does this error show up during normal operation? E.g. during examine() or something?

Reply:
Addressed.

> Does this error show up during normal operation? E.g. during examine() or something?

I've added the message just in case. Have not observed it yet.